### PR TITLE
Update text, code and images

### PR DESCRIPTION
**Pull request by jansule**
- Updated descriptions to have clear tasks
- Update code and example app layout
- Update screenshots
- Replace `Titlebar` with antd's `Drawer` component for a more appealing GUI

### CODE REVIEW

File: gitbook/higher-order-components/mappify.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 * The `mappify` HOC grabs a map object from the context (see also [here](https://reactjs.org/docs/context.html)) and passes it as a prop to the wrapped component
 * Commonly used in combination with the `MapProvider`
 
+**Task:** Update you app by mappifying your components.
```
Comment (marcjansen) [inline gitbook/higher-order-components/mappify.md]:
```suggestion
**Task:** Update your app by mappifying your components.
```

**Comment by jansule** [comment]
> Iam wondering why we need all the
> 
> ```js
> map.on('postcompose', map.updateSize);
> ```
> 
> calls. This looks a bit suspicious.

@dnlkoch  and I did this as a quickfix for forcing an updateSize. Otherwise the map is stretched and fonts are blurry. This is probably a bug in react-geo.

File: gitbook/higher-order-components/mappify.md (hunk 1 above)
Comment (jansule) [reply gitbook/higher-order-components/mappify.md]: fixed in https://github.com/terrestris/react-geo-ws/pull/15/commits/5f38d242d360fc0af136f4d799f4783275692b80

**Comment by dnlkoch** [comment]
Nice work!

One remark only: Could you provide links to the react-geo documentation for all components handled in the workshop? I think this is of greater interest as the linked source code.

**Comment by dnlkoch** [comment]
> LGTM
> 
> Iam wondering why we need all the
> 
> ```js
> map.on('postcompose', map.updateSize);
> ```
> 
> calls. This looks a bit suspicious.

Yes, it is suspicious, but currently needed (see comment ahead). I'll give it another try to remove it…

**Comment by jansule** [comment]
> Nice work!
> 
> One remark only: Could you provide links to the react-geo documentation for all components handled in the workshop? I think this is of greater interest as the linked source code.

I will do so as soon as https://github.com/terrestris/react-geo/issues/1072 was fixed. Currently the examples only show an error log.

**Pull request merged**